### ISSUE TITLE
Password Reset missing

### ISSUE THREAD

**Issue by Tonomis** [hrfee/jfa-go#113]
I just update to v0.3.6 this morning, where is Password Reset setting ?

Am I just stupid ?

![image](https://user-images.githubusercontent.com/63344585/120314089-263b9f00-c2db-11eb-8798-355d0e289726.png)

**Comment by hrfee** [comment]
Nope, This looks like a  bug. Could you try disabling then enabling messages/notifications? If its still hidden, could you pull up the javascript console (Right-click > Inspect Element > Console), then refresh the page? If there's any errors, take a screenshot and send them.

**Comment by Tonomis** [comment]
Enabling Messages/notifications make it back, thanks !
(but it's disappearing when I disable it)

**Comment by hrfee** [comment]
That's intentional, if you disable messages it can't send the PIN to users.
